tests/test_execution.py

```diff
@@ -245,3 +245,124 @@ def test_composite_traintuples_execution(factory, session):
     assert set([composite_traintuple_1.key, composite_traintuple_2.key]).issubset(
         composite_traintuple_keys
     )
+
+
+def test_aggregate_traintuples(factory, session):
+    """Aggregate traintuples."""
+    number_of_traintuples_to_aggregate = 3
+    spec = factory.create_dataset()
+    dataset = session.add_dataset(spec)
+
+    train_data_samples = []
+    for i in range(number_of_traintuples_to_aggregate):
```

Comment:
My linter lights up because of this unused `i`. Could we use `_` instead?

Reply:
Done